### ISSUE TITLE
Accessibility bug: Configuration of combined modules is missing labels

### ISSUE THREAD

**Issue by pitermach** [exelban/stats#2322]
This is the second accessibility issue I discovered when trying to use Stats with VoiceOver. In general, most configuration screens work well and every control you need is clearly labeled. The exception to this is the combined module configuration. When you switch to the settings page, and turn on the combine modules, switch, try to navigate through the interface with CTRL+Option+Right and left. Notice that every module is just read as “image”, with the exception of one which is labeled as “TV” for some reason.

I was able to figure out you can rearrange the order with mouse dragging, but not if it’s possible to decide which modules should be a part of the combined item other than just turning them off in the global settings. If it’s possible to configure this here, then I think it would be great if apart from the modules just being labeled so you know what’s what, performing the VoiceOver press command (CTRL+Option+Space) would toggle a module from being included or not in the combined section, because dragging items around with VoiceOver is complicated and error-prone.

Device Info: Mac OS 15.2 on M1 and M3 pro Macs.

**Comment by exelban** [comment]
I will go throw the whole application in the #2321 and will try to add labels to all elements that could be visible.
